.github/workflows/mobile-ios_build.yml

```diff
@@ -137,34 +143,21 @@ jobs:
       matrix:
         include:
         - name: Build swift baseline app
-          args: >-
-            build
-            --config=mobile-remote-ci-macos-ios
-            //test/swift/apps/baseline:app
           app: //test/swift/apps/baseline:app
           expected-status: 301
           target: swift-baseline-app
         - name: Build swift experimental app
           args: >-
-            build
             --config=mobile-remote-ci-macos-ios-admin
-            //test/swift/apps/experimental:app
           app: //test/swift/apps/experimental:app
           expected-status: 200
           target: swift-experimental-app
         - name: Build swift async await
-          args: >-
-            build
-            --config=mobile-remote-ci-macos-ios
-            //examples/swift/async_await:app
           app: //examples/swift/async_await:app
-          expected-status: 200
+          expected: >-
+            \[2\] Uploaded 7 MB of data
```

Comment:
Not sure I follow, is this reading the logs and making sure this statement is found, as opposed to looking at the status code?  Is it because sometimes failures still report a 200?  Should the test itself be fixed to not report 200 when it fails?

Reply:
this is what the original test did